cli/pull.go

```diff
@@ -133,5 +133,14 @@ func parseNameTag(input string) (string, string) {
 		tag = fields[1]
 	}
 
+	if strings.Contains(name, "/") {
+		count := strings.Count(name, "/")
+		if count == 1 {
+			name = "docker.io/" + name
```

Comment:
Actually hard code is not our first choice. Maybe we can set a const variable there to represent this.